Gemfile.lock

```diff
@@ -41,19 +41,19 @@ GEM
     addressable (2.7.0)
       public_suffix (>= 2.0.2, < 5.0)
     arel (8.0.0)
-    bcrypt (3.1.14)
+    bcrypt (3.1.15)
```

Comment:
I ran a `bundle update` and it updated these gems

Reply:
- [x]

Reply:
a-ok